### PR TITLE
Revert CI workflow (1 of 2)

### PR DESCRIPTION
**Pull request by eradman**
## What type of PR is this? 

- [x] Revert

## Description

Commands used

```
rm -rf .github/
cp -r ../redash-alt/.github .

git add .github/workflows/ci.yml .github/workflows/preview-image.yml

git commit -m 'Revert CI workflow'
```

## How is this tested?

- [x] Unit tests (pytest, jest)
- [x] E2E Tests (Cypress)
- [x] Manually

## Related Tickets & Documents

https://github.com/getredash/redash/discussions/6964

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,15 +23,18 @@ jobs:
...
+      - run: sudo pip install black==23.1.0 ruff==0.0.287
       - run: ruff check .
       - run: black --check .
 
   backend-unit-tests:
     runs-on: ubuntu-22.04
     needs: backend-lint
     env:
-      FRONTEND_BUILD_MODE: 0
+      COMPOSE_FILE: .ci/compose.ci.yaml
```
Comment (justinclift) [inline .github/workflows/ci.yml:34]: Missed a sub-directory? :smile:

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -135,24 +117,21 @@ jobs:
...
-          cd viz-lib
-          yarn test
+        run: cd viz-lib && yarn test
       - run: yarn lint
 
   frontend-e2e-tests:
     runs-on: ubuntu-22.04
     needs: frontend-lint
     env:
+      COMPOSE_FILE: .ci/compose.cypress.yaml
```
Comment (justinclift) [inline .github/workflows/ci.yml:131]: As above.

**Comment by justinclift** [comment]
Aside from the `.ci` subdirectory query, the rest looks workable. :smile:

File: .github/workflows/ci.yml (hunk 1 above)
Comment (eradman) [reply .github/workflows/ci.yml:34]: Ah, no. I am staging this as two separate commits. I'll post the second so you can see what I'm trying to do.

Comment (justinclift) [reply .github/workflows/ci.yml:34]: No worries. :smile:

**Review by justinclift — APPROVED** [review]
Cool, looks good to me.

Thanks @eradman. :smile:

**Pull request merged**